spec/index.html

```diff
@@ -1694,6 +1694,8 @@ <h2>Changes between RDF 1.1 and RDF 1.2</h2>
     <li>Added the <a>base direction</a> element as part of 
       a <a>literal</a>,
       and a description of its use in <a href="#section-text-direction" class="sectionRef"></a>.</li>
+    <li>Changed reference from DOM4, which was not a recommendation at the time, to [[DOM]],
+      making the <a>rdf:HTML</a> and <a>rdf:XMLLiteral</a> datatypes normative.</li>
```

Comment:
"normative" - that has two readings (1) the definition is normative as possible recognized dadatypes (2) the spec makes them normative/required.

```suggestion
      making the definitions of <a>rdf:HTML</a> and <a>rdf:XMLLiteral</a> datatypes normative.</li>
```
(given other discussions, we need to clear that the RDF Data Model does not mandate implementation and all the consequence of these or any other datatypes)

Reply:
@afs, this slipped before subsequent update; I've added the change you suggested, but any other discussion about not mandating implementation of these datatypes probably needs to go in another issue/PR.

Reply:
Thank you for adding the change.

RDF 1.1 did not mandate implementation of any datatypes (except, arguably, `rdf:langString`).